### PR TITLE
RED-22899 Add instructions for CRDB flush

### PR DESCRIPTION
**Pull request by bmansheim**

### CODE REVIEW

File: content/rc/how-to/flush-db.md [hunk 1]
```diff
@@ -35,4 +35,12 @@ includes all data in memory and persisted to disk. So before doing
 flushall, it is best practice to [backup your
 database]({{< relref "/rc/administration/configure/backups.md" >}}) first.
 
-### 
+### Flushing Redis Enterprise Cloud CRDBs
+
+When you flush a CRDB, multiple resources flush their data simultaneously.
```
Comment (Paz-Yanover) [inline content/rc/how-to/flush-db.md]: 1. I didn't understand what the following sentence means: "When you flush a CRDB, multiple resources flush their data simultaneously". Maybe it worth saying that flush a CRDB will remove / erase all the data over all replicas with no way to recover it (unless you have a backup)
2. Please add also the possibility to flush the CRDB using the REST API (as written in ticket- RED-25549 by Yossi)

Comment (bmansheim) [reply content/rc/how-to/flush-db.md]: 2. I added the API instructions below

Comment (Paz-Yanover) [reply content/rc/how-to/flush-db.md]: 1. Sounds good to me.

File: content/rc/how-to/flush-db.md [hunk 2]
```diff
@@ -40,7 +40,12 @@ database]({{< relref "/rc/administration/configure/backups.md" >}}) first.
...
 
-To flush a CRDB:
+To flush a CRDB from crdb-cli:
 
 1. To find the ID of the CRDB, run: `crdb-cli crdb list`
-1. To flush the CRDB, run: `crdb-cli crdb flush --crdb-guid <ID_of_the_CRDB>`
\ No newline at end of file
+1. To flush the CRDB, run: `crdb-cli crdb flush --crdb-guid <ID_of_the_CRDB>`
+
+To flush a CRDB from the API:
```
Comment (Paz-Yanover) [inline content/rc/how-to/flush-db.md]: Maybe: To flush a CRDB using the REST API:

File: content/rc/how-to/flush-db.md [hunk 3]
```diff
@@ -40,7 +40,12 @@ database]({{< relref "/rc/administration/configure/backups.md" >}}) first.
...
+
+1. To find the ID of the CRDB, run: `curl -v -u <user>@<password> -X PUT https://<cluster-fqdn>:9443/v1/crdbs/`
```
Comment (Paz-Yanover) [inline content/rc/how-to/flush-db.md]: Please check the ticket (https://redislabs.atlassian.net/browse/RED-25549) again and take the updated command that Alon (will) add.

File: content/rc/how-to/flush-db.md [hunk 4]
```diff
@@ -35,4 +35,41 @@ includes all data in memory and persisted to disk. So before doing
 flushall, it is best practice to [backup your
 database]({{< relref "/rc/administration/configure/backups.md" >}}) first.
 
-### 
+### Flushing Redis Enterprise Cloud CRDBs
```
Comment (Paz-Yanover) [inline content/rc/how-to/flush-db.md]: Why Cloud?

File: content/rc/how-to/flush-db.md [hunk 5]
```diff
@@ -35,4 +35,41 @@ includes all data in memory and persisted to disk. So before doing
...
+
+When you flush a CRDB, multiple resources flush their data simultaneously.
```
Comment (Paz-Yanover) [inline content/rc/how-to/flush-db.md]: Maybe it worth mentioning the word "replicas" as well- so the data is flushed over all replicas

File: content/rc/how-to/flush-db.md [hunk 6]
```diff
@@ -35,4 +35,41 @@ includes all data in memory and persisted to disk. So before doing
...
+    ```src
+    crdb-cli crdb list
+    ```
+
+1. To flush the CRDB, run:
+
+    ```src
+    crdb-cli crdb flush --crdb-guid <ID_of_the_CRDB>
+    ```
+
```
Comment (Paz-Yanover) [inline content/rc/how-to/flush-db.md]: Don't we want here to have a command line for tracking the task status or the crdb-cli is not mature enough in this respect?

File: content/rc/how-to/flush-db.md [hunk 7]
```diff
@@ -35,4 +35,41 @@ includes all data in memory and persisted to disk. So before doing
...
+### Flushing Redis Enterprise Cloud CRDBs
+
+When you flush a CRDB, multiple resources flush their data simultaneously.
+To flush data from a CRDB, use the flush command in the crdb-cli.
+
+1. To find the ID of the CRDB, run:
+
+    ```src
+    crdb-cli crdb list
+    ```
```
Comment (Paz-Yanover) [inline content/rc/how-to/flush-db.md]: Maybe it worth printing example of the output and where the ID is located

File: content/rc/how-to/flush-db.md [hunk 8]
```diff
@@ -35,4 +35,41 @@ includes all data in memory and persisted to disk. So before doing
...
+    curl -v -u <user>@<password> -X PUT https://<cluster-fqdn>:9443/v1/crdbs/
+    ```
+
+1. To flush the CRDB, run:
+
+    ```src
+    curl -v -u <username>:<password> -X PUT https://<cluster-fqdn>:9443/v1/crdbs/<guid>/flush
+    ```
+
+    The command output contains the GUID of the flush task.
```
Comment (Paz-Yanover) [inline content/rc/how-to/flush-db.md]: Is it called GUID? I don't think so

File: content/rc/how-to/flush-db.md (hunk 4 above)
Comment (bmansheim) [reply content/rc/how-to/flush-db.md]: Moved the CRDB commands to RS

File: content/rc/how-to/flush-db.md (hunk 8 above)
Comment (bmansheim) [reply content/rc/how-to/flush-db.md]: Right, task ID

File: content/rc/how-to/flush-db.md (hunk 6 above)
Comment (bmansheim) [reply content/rc/how-to/flush-db.md]: Added

File: content/rc/how-to/flush-db.md (hunk 5 above)
Comment (bmansheim) [reply content/rc/how-to/flush-db.md]: all of the replicas flush their data simultaneously

File: content/rc/how-to/flush-db.md (hunk 7 above)
Comment (bmansheim) [reply content/rc/how-to/flush-db.md]: OK

**Pull request merged**